extensions/bitwarden/src/search.tsx

```diff
@@ -145,6 +153,16 @@ function ItemListItem(props: { item: Item; folder: Folder | undefined; refreshIt
 
   const tree = treeify.asTree(cleanItem, true, false);
 
+  async function copyTotp(id: string) {
```

Comment:
Why not adding all the logic into the loadTotp function ? I feel like splitting the logic between the two functions make the code harder to grasp.

Reply:
Thanks @pomdtr - I agree! I haven't written react for a few years now so I wasn't sure of any security issues with passing the session token as a prop, so I tried to remove that by wrapping it in a closure. But if its fine to do, grouping it into one function is definitely better IMO